checkers/mozlenium/Dockerfile

```diff
@@ -76,7 +77,7 @@ RUN wget https://chromedriver.storage.googleapis.com/2.41/chromedriver_linux64.z
 
 ENV NODE_PATH /app/node_modules
 
-COPY wrapper.js package.json mozlenium.js firefox-browser.js chrome-browser.js entrypoint.sh wrapper.js /app/
+COPY package.json mozlenium.js firefox-browser.js chrome-browser.js entrypoint.sh wrapper.js /app/
```

Comment:
You don't wanna copy the same file twice?

Reply:
lol